src/setup_payload/QRCodeSetupPayloadParser.cpp

```diff
@@ -376,13 +376,14 @@ CHIP_ERROR QRCodeSetupPayloadParser::populatePayload(SetupPayload & outPayload)
     static_assert(kProductIDFieldLengthInBits <= 16, "Won't fit in uint16_t");
     outPayload.productID = static_cast<uint16_t>(dest);
 
-    err = readBits(buf, indexToReadFrom, dest, kCustomFlowRequiredFieldLengthInBits);
+    err = readBits(buf, indexToReadFrom, dest, kCommissioningFlowFieldLengthInBits);
     SuccessOrExit(err);
-    static_assert(kCustomFlowRequiredFieldLengthInBits <= 8, "Won't fit in uint8_t");
-    outPayload.requiresCustomFlow = static_cast<uint8_t>(dest);
+    static_assert(kCommissioningFlowFieldLengthInBits <= 8, "Won't fit in uint8_t");
+    outPayload.commissioningFlow = static_cast<CommissioningFlow>(dest);
```

Comment:
Shouldn't we now be asserting that `CommissioningFlow` has enough bits to fit this?  Where does `uint8_t` come in?

Reply:
outPayload.commissioningFlow has 8 bits. I will update the static assert message though.